Tmain/output-encoding-option.d/tags-e

```diff
@@ -0,0 +1,13 @@
+!_TAG_FILE_ENCODING	shift_jis	//
```

Comment:
I don't understand why this file is needed.
Maybe you may want to make another pull request and this file may be part of it.